introduction/modules/modules.md

```diff
@@ -83,21 +86,21 @@ document. This does *not* generate an actual document, but writes the calls to t
 
 ### Local mail
 
-The local mail module is a stub implementation of the mail module. It is used as a local test implementation that 
-allows the application to start without having to provide an actual implementation. This does *not* send an actual 
-mail.
+The local email module is a stub implementation of the module called 'Mail'. It is used as a local test implementation 
+that allows the application to start without having to provide an actual implementation. This does *not* actually send 
+emails.
 
 ### Mail
 
-Mail describes how valtimo sends mail in a consistent way. Mails are filtered based on configured filters. For example, 
-mail to blacklisted addresses can be filtered out. Dispatching the email after filtering is facilitated by another 
-module.
+Mail describes how Valtimo sends emails in a consistent way. Emails are filtered based on configured filters. For 
+example, emails to blacklisted addresses can be filtered out. Dispatching the email after filtering is facilitated by 
+another module.
 
 ### Mandrill
 
-Mailchimp is a SaaS solution for sending email. Mailchimp is the name of the 
+Mailchimp is a SaaS solution for sending emails. Mailchimp is the name of the 
 [transactional mail addon](https://mailchimp.com/features/transactional-email/) that Mailchimp provides. The Valtimo
-Mandrill module provides a mail implementation that uses Mandrill to send mail.
+Mandrill module provides an email implementation that uses Mandrill to send emails.
```

Comment:
`Mandrill module provides an mail implementation that uses Mandrill to send emails.`

Since the module is called mail, it should be mail implementation.